pkg/scte35/stream.go

```diff
@@ -0,0 +1,354 @@
+// Copyright 2022 Adrian of Doom
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or   implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+
+package scte35
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+const (
+    // pktSz is the size of an MPEG-TS packet in bytes.
+    pktSz = 188
```

Comment:
We try to avoid unnecessary abbreviations in function/method/variable names unless there is a really good reason to do so (the variable is long and clunky and the abbreviation is obvious). 

e.g. `packetSize`, `bufferSize`, etc